packages/taquito/version-stamping.js

```diff
@@ -0,0 +1,39 @@
+const { resolve, relative } = require('path');
+const { writeFileSync } = require('fs-extra');
+const { version } = require('./package.json');
```

Comment:
Won't this cause getVersion to show the version tag of the last released version all the time? If we are building a preview build from CI, having the branch name in the `VersionInfo.version` property would be reasonable, and that would clearly signal if the build is a preview, or an official build.

Reply:
If I remember correctly, for the pre-release packages published from the CI, version will properly reflect the branch.
But exposing the branch name is a great idea anyway

Reply:
I checked on preview builds repository for this PR build it has right version in version.js file i.e. 8.1.0-94548f5e--781. I will check final build version and comment back.

```
exports.VERSION = {
    "commitHash": "94548f5e6b265091bf7c278ded9a866eb3f21152",
    "version": "8.1.0-94548f5e--781"
};

Reply:
I created a dummy version "9.1.2" and followed steps for final release as per notion till part where we push to master/npm. 

I checked packages and all had right version.

```
export const VERSION = {
    "commitHash": "9b638f88e8ca6b2716c2a49462a525b35012804b",
    "version": "9.1.2"
};